kubetest2-gce/deployer/common.go

```diff
@@ -129,10 +129,14 @@ func (d *deployer) buildEnv() []string {
 	// the run of kubetest2 and so should be placed in the artifacts directory
 	env = append(env, fmt.Sprintf("KUBECONFIG=%s", d.kubeconfigPath))
 
-	// kube-up and kube-down get this as a default ("kubernetes") but log-dump
-	// does not. opted to set it manually here for maximum consistency
+	// kube-up and kube-down get this as a default ("kubernetes" / "e2e-test-${USER}")
+	// but log-dump does not, set it explicitly here for maximum consistency
 	env = append(env, fmt.Sprintf("KUBE_GCE_INSTANCE_PREFIX=%s", d.instancePrefix))
 
+	// kube-up and kube-down get this as a default ("default" / "e2e-test-${USER}")
+	// but log-dump does not, set it explicitly here for maximum consistency
```

Comment:
neat. did kubetest do this? are there more of these?

Reply:
> did kubetest do this ?

I don't think so it just relied on the defaults.

> are there more of these?

Hopefully not 😅